### PR TITLE
Add Zombocam blog post

### PR DESCRIPTION
**Pull request by cristeahub**

### CODE REVIEW

File: _posts/2017-01-05-enhancing-your-webcam.md [hunk 1]
```diff
@@ -0,0 +1,100 @@
+---
+layout: post
+title:  "Enhancing your webcam using canvas.captureStream()"
+date:   2017-01-05 12:00:00
+author: "Zombocam"
```
Comment (sigvef) [inline _posts/2017-01-05-enhancing-your-webcam.md]: Afaict the convention on this blog is to use the name of the author here (as opposed to the "marketing entity").

Comment (cristeahub) [reply _posts/2017-01-05-enhancing-your-webcam.md]: So who wrote the original post again?

Comment (sigvef) [reply _posts/2017-01-05-enhancing-your-webcam.md]: Is it the medium post? Then it was me :3

Comment (cristeahub) [reply _posts/2017-01-05-enhancing-your-webcam.md]: Alright, will update! :)

**Comment by cristeahub** [comment]
ping

File: _posts/2017-01-05-enhancing-your-webcam.md [hunk 2]
```diff
@@ -0,0 +1,102 @@
...
+That means properly forwarding all sorts of properties on the Streams and Tracks returned from getUserMedia that applications might expect and depend on.
+
+One specific example of this that we ran into was with [Appear.in’s new premium offering](https://appear.in/information/premium/), where you can screen-share and show your webcam stream in your meeting room at the same time.
+The application relied on the name of one of the Tracks to be “Screen”, which we didn’t properly forward to our Tracks that we got from our canvas.
+Because of this, Appear.in didn’t know which of the tracks was the screen-sharing track, and things stopped working.
+Properly forwarding the name property solved the issue, and we learned an important lesson in the virtues of actually being transparent when trying to transparently intercept APIs.
+
+# What's next: audio filters
+
+With the new release of [Zombocam](https://www.zombocam.com) we’ve week taken it one step further and enhanced getUserMedia audio tracks as well using the [Web Audio API](https://developer.mozilla.org/en-US/docs/Web/API/Web_Audio_API).
```
Comment (stianjensen) [inline _posts/2017-01-05-enhancing-your-webcam.md]: With the new release of [Zombocam](https://www.zombocam.com) **we’ve week** taken

File: _posts/2017-01-05-enhancing-your-webcam.md [hunk 3]
```diff
@@ -0,0 +1,100 @@
...
+Productivity will never be the same.
+
+Since effects can be chained in Zombocam, the output from one effect’s rendering pass is fed directly as input to the next effect’s rendering pass.
+This opens for a wide array of different possible effect combinations.
+
+
+<img src="/images/zombocam1.png" alt="Zombocam can turn you into a cyclops if you’re not careful when chaining effects!"/>
+<br/>
+
+# Works everwhere! (\*)
```
Comment (stianjensen) [inline _posts/2017-01-05-enhancing-your-webcam.md]: s/everwhere/everywhere/

**Comment by cristeahub** [comment]
ping

**Pull request merged**